content/en/users/datahub/api/_index.md

```diff
@@ -53,10 +53,89 @@ The following variables should be set:
 ONEZONE_HOST=https://datahub.egi.eu
 OIDC_TOKEN=<OIDC_ACCESS_TOKEN>
 curl -H "X-Auth-Token: egi:$OIDC_TOKEN" -X POST \
-  -H 'Content-type: application/json' -d '{}' \
-  "$ONEZONE_HOST/api/v3/onezone/user/client_tokens"
+  -H 'Content-type: application/json'  \
+  "$ONEZONE_HOST/api/v3/onezone/user/tokens/named" -d '{ \
+  "name": "REST and CDMI access token", \
+  "type": { \
+    "accessToken": {} \
+  }, \
+  "caveats": [ \
+    { \
+      "type": "interface", \
+      "interface": "rest" \
+    } \
+  ] \
+}'
```

Comment:
Are you sure about all those `\` at the EOL  for the data sent via CURL?

Reply:
in my shell it works:-)

Reply:
nope sorry, you are right, need to remove the `\`